### ISSUE TITLE
[lion-input-stepper] Value change is not announced by screen reader (Chrome + NVDA)

### ISSUE THREAD

**Issue by riovir** [ing-bank/lion#1894]
At the time of writing, using the latest Chrome (v109.0.5414.120) and NVDA (2022.4.0.27401) with Tools > Speach Viewer on, open https://lion-web.netlify.app/components/input-stepper/overview/ the tab to the LionInputStepper. Press the up arrow key 3 times. The stepper will correctly show 3.

## Expected behavior

The screen reader / speech viewer also describes the component and more importantly announces the 1, 2, 3 values as the arrow key is pressed.

## Actual Behavior

No announcement happens. The user seemingly is left guessing the value of the component.

## Additional context

Repeated the same test with Firefox. The screen reader correctly describes the component and announces the values as well. 
At the time of reporting it's unclear what makes Chrome special.

**Comment by Sciurus7** [comment]
I actually know what is causing the bug. The issue is that role="spinbutton" is on the <lion-input-stepper>. When removing that and setting it on the input nod instead it works as expected. I verified this using the dev tools to move the role and testing with NVDA (v2022.4.0.27401) and Chrome (v109.0.5414.75)

**Comment by gerjanvangeest** [comment]
@riovir thanks for the issue
@Sciurus7 thanks for the fix!
Is any of you also willing to contribute it via a PR?

**Comment by Sciurus7** [comment]
I can create a PR on Friday. (Need to focus on my sprint work for today and tomorrow.)

**Comment by tlouisse** [comment]
> I can create a PR on Friday. (Need to focus on my sprint work for today and tomorrow.)

Any progress so far @Sciurus7?
